en: Update doc for aurora migration (#377)
pingcap/docs-dm: en/hardware-and-software-requirements.md

>
> - In the test environment, DM-master and DM-worker used for functional verification can be deployed on the same server.
> - To prevent interference with the accuracy of the performance test results, it is **not recommended** to use low-performance storage and network hardware configurations.

> > - If you need to verify the function only, you can deploy a DM-master on a single machine. The number of DM-worker deployed is equivalent to the number of upstream MySQL instances at least. To ensure high availability, it is recommended to deploy more DM-workers.

Reviewer: ```suggestion
> - If you need to verify the function only, you can deploy a DM-master on a single machine. The number of DM-worker deployed must be greater than or equal to the number of upstream MySQL instances. To ensure high availability, it is recommended to deploy more DM-workers.
```
